packages/lib/src/components/ApplePay/ApplePay.tsx

```diff
@@ -78,7 +78,7 @@ class ApplePayElement extends UIElement<ApplePayElementProps> {
                         this.setState({ applePayToken: btoa(JSON.stringify(event.payment.token.paymentData)) });
                     }
 
-                    this.onSubmit();
+                    super.submit();
```

Comment:
Why the change to explicitly call `super`?

Reply:
There is already a `submit` method declared in this component. In order to call the one of the parent component, I had to use `super`.

Reply:
Ok, it's just that it used to call `super.onSubmit()` directly (albeit via the subclass' `this.onSubmit()`) but now it calls `super.submit()` which first performs validation before it calls `this.onSubmit` in the `UIElement`.
It makes sense to validate first - but I just wanted to check on the slight change in behaviour

Reply:
I get it. As I defined the interface of the `UIElement`, the `onSubmit` isn't available anymore on the subclasses since it is a private method. It is better to force the components to go through the `submit` method , which will validate before doing the actual request.

Reply:
ok, cool. thanks for the explanation :-)